### PR TITLE
HAPP-332 | build and run local docker containers

### PR DESCRIPTION
**Pull request by sunghwan-park-fw**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM node:16-alpine
```
Comment (hitankar-freshworks) [inline Dockerfile:1]: This will make the build time massive. I suggest we do a separate Dockerfile for each app.

Comment (sunghwan-park-fw) [reply Dockerfile:1]: Only when we have a change in the dependencies, we need to build again. When it builds, it installs all dependencies for web and api and reuse the image for both. If we separate the installation and images, it couldn't make the time shorter than this. Also I'm not sure how we can separate it in the yarn workspaces.

Comment (sunghwan-park-fw) [reply Dockerfile:1]: How long does it take in your env? it took just a few minutes in my env.

Comment (hitankar-freshworks) [reply Dockerfile:1]: https://github.com/bcgov/myhealthBC-symptom-checker/pull/11

**Review by hitankar-freshworks — APPROVED** [review]

**Pull request merged**